src/Export/Adapters/UserGroupsAdapter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Adapters;
+
+use FINDOLOGIC\Export\Data\Usergroup;
+use FINDOLOGIC\FinSearch\Export\ExportContext;
+use FINDOLOGIC\FinSearch\Utils\Utils;
+use Shopware\Core\Checkout\Customer\Aggregate\CustomerGroup\CustomerGroupEntity;
+use Shopware\Core\Content\Product\ProductEntity;
+
+class UserGroupsAdapter
+{
+    /**
+     * @return Usergroup[]
+     */
+    public function adapt(ProductEntity $product, ExportContext $exportContext): array
```

Comment:
Please include the export context via constructor parameter, like it is done in the `AttributeAdapter`.